synth-publish.go

```diff
@@ -220,8 +221,41 @@ func checkPublishingStatus(latest postedData, result chan<- publicationResult, s
 
 	switch resp.StatusCode {
 	case http.StatusOK:
+		cmdR := exec.Command("kubectl", "delete", "cm", "synthetic-image-alarm", "--ignore-not-found")
```

Comment:
Is the `synthetic-image-alarm` configmap different from the `synthetic-tid` one that is being handled in the `http.StatusNotFound` case?

Reply:
synthetic-image-alarm implements a state for the alarm if the alarm was already fired once and not resolved. The state is used to suppress the alarm in case of many invocations - i.e. the alarm will trigger and will check the state and the state is true it will exit and do nothing. This is to avoid flooding of #upp-prod-incindents channel.

Reply:
Ok, how is this configmap created (I see only that it is being deleted)?

Reply:
It is created by the image trace job here https://github.com/Financial-Times/coco-image-trace/blob/master/alarm.sh#L26